src/main/resources/assets/trafficcontrol/lang/en_us.lang

```diff
@@ -34,21 +35,40 @@ tile.trafficcontrol.traffic_sensor_right.name=Traffic Sensor (Right)
 item.trafficcontrol.crossing_relay_box.name=Crossing Gate Relay
 item.trafficcontrol.crossing_relay_tuner.name=Tuner
 item.trafficcontrol.traffic_light_frame.name=Traffic Light Frame
+
 tile.trafficcontrol.street_sign.name=Street Sign
 item.trafficcontrol.traffic_light_5_frame.name=Traffic Light Frame (5 Bulb)
+
 item.trafficcontrol.traffic_light_doghouse_frame.name=Traffic Light Frame (Doghouse)
+
 item.trafficcontrol.traffic_light_1_frame.name=Traffic Light Frame (One Bulb)
+
 item.trafficcontrol.traffic_light_2_frame.name=Traffic Light Frame (Two Bulbs)
+
 item.trafficcontrol.traffic_light_4_frame.name=Traffic Light Frame (Four Bulbs)
+
+item.trafficcontrol.traffic_light_6_frame.name=Traffic Light T Frame (Four Bulbs)
+
```

Comment:
All the extra spaces introduced through this PR make it hard to tell where the sections start and end